server/backend/api/alembic/versions/V19_update_fom_config_data.py

```diff
@@ -0,0 +1,235 @@
+"""update_fom_config_data
+
+Revision ID: V19
+Revises: V18
+Create Date: 2023-01-18 22:56:44.678454
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import sqlmodel
+
+
+
+# revision identifiers, used by Alembic.
+revision = 'V19'
+down_revision = 'V18'
+branch_labels = None
+depends_on = None
+
+client_id_dev_fom_oidc_client = '1a8pkq0psq0daj5e6ir3ppcjkj'
+client_id_test_fom_oidc_client = '7b6eki43nahus9ca0lhjs6m568'
+client_id_prod_fom_oidc_client = '1rhdfiek5ntmk2kg39d6e31p46'
+
+migrate_sql_str = f"""
+UPDATE app_fam.fam_application SET (application_name, application_description, update_user, update_date) =
+	('fom_dev', 'Forest Operations Map (DEV)', CURRENT_USER, CURRENT_DATE)
+WHERE application_name = 'fom'
+;
+
+UPDATE app_fam.fam_role SET (role_name, role_purpose, update_user, update_date) =
+	('FOM_DEV_ACCESS_ADMIN', 'Provides the privilege to assign or unassign all roles for FOM (DEV)', CURRENT_USER, CURRENT_DATE)
+WHERE role_id =
+	(SELECT role_id FROM app_fam.fam_role WHERE role_name = 'FOM_ACCESS_ADMIN')
+;
+
+INSERT INTO app_fam.fam_application (
```

Comment:
It doesn't matter (no need to change), but just for reference these two inserts could have been combined together into one statement.

Reply:
Inserts combined now.